gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceExtractor.java

```diff
@@ -598,6 +595,8 @@ public String getTimestampPredicateCondition(String column, long value, String v
     String[] batchIdResultIdArray = partitionPkChunkingBatchIdResultIterator.next().split(":");
     String batchId = batchIdResultIdArray[0];
     String resultId = batchIdResultIdArray[1];
+    log.info(String.format("PK-Chunking work unit: fetching file for (jobId=%s, batchId=%s, resultId=%s) ",
```

Comment:
Please use {} logger format instead of String.format.

Reply:
BTW, we were using this way a lot though. I copied the code :)
I did some searching in our code, I didn't find good example. Can you pint me out a sample code? Are you talking about `MessageFormat`?

Reply:
thanks for online talk, fixed.

Reply:
thanks for offline talk, fixed. :)